.github/workflows/ci-infra.yml

```diff
@@ -26,6 +26,17 @@ jobs:
         run: |
           echo "If this fails, run 'make infra-format'"
           make infra-lint
+  validate-terraform:
```

Comment:
you might want to set a lock-timeout on your commands so jobs don't fail immediately if there are two jobs/PRs running simultaneously.

Otherwise, the first job will grab the tf lock and the second job will immediately fail when trying to grab the lock.

```
validate-terraform:
    env:
      TF_CLI_ARGS: '-lock-timeout=120s'
```

Reply:
is that true? i thought `terraform validate` doesn't actually need the backend. make infra-validate-module does this:

```
	terraform -chdir=infra/modules/$* init -backend=false
	terraform -chdir=infra/modules/$* validate
```

and from the [terraform validate docs](https://www.terraform.io/cli/commands/validate):

> The terraform validate command validates the configuration files in a directory, referring only to the configuration and not accessing any remote services such as remote state, provider APIs, etc.

that said if i'm wrong happy to add it. i'll go ahead and merge for now and add it in a separate PR if we need it

Reply:
ah you're right 🙃